scripts/linux/runE2ETest.sh

```diff
@@ -352,6 +356,18 @@ function process_args() {
         elif [ $saveNextArg -eq 29 ]; then
             EVENT_HUB_CONSUMER_GROUP_ID="$arg"
             saveNextArg=0
+        elif [ $saveNextArg -eq 30 ]; then
+            ANALYZER_LA_ENABLED_STRING="$arg"
```

Comment:
why put "_STRING" here?
```suggestion
            ANALYZER_LA_ENABLED="$arg"
```

Reply:
The `"_STRING"` is intentionally there to prevent the confusion if it should be `false` or `"false"` -- it should be `"false"`